stack-9.10.yaml

```
@@ -0,0 +1,15 @@
+# Copyright 2019 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+resolver: nightly-2024-04-06
```

```suggestion
resolver: nightly-2024-10-28
```

Maybe use the newest since we are carving something custom anyway?